src/SFA.DAS.Admin.Aan.Web/Views/MemberProfile/_MemberActivities.cshtml

```diff
@@ -3,7 +3,26 @@
 @model MemberActivitiesViewModel;
 
 <div>
-    <h3 class="govuk-heading-m">Upcoming events (@Model.FutureEventsCount)</h3>
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-2">@Model.FirstName's activities</h1>
+    <p>Last event sign up: @Model.LastEventSignUpDate</p>
+    <h3 class="govuk-heading-m">
+        In the past 12 months
```

Comment:
Ticket content reads "In the last 12 months"

EDIT Looks like the ticket is inconsistent between "last 12" and "past 12", worth double checking

Reply:
We normally followed figma when there is contradiction between the ticket and figma.